src/navigation/components/profile/ProfileDropdown.jsx

```diff
@@ -23,10 +24,27 @@ export const ProfileDropdownComponent = ({
 	profile,
 	logout,
 	groups,
+	groupDraft,
 	groupHome,
 	allGroupsLabel,
 	allGroupsLink,
 }) => {
+	const groupDraftsContent = groupDraft => (
```

Comment:
Instead of redefining this component on every render, prefer making it a pure function that takes `(groupDraft, groupHome)` args.

Even better, make it a function component

```jsx
const GroupDraftItem = ({ groupDraft, groupHome }) => (
  <li>...</li>
)

// ...

<ul className="list">
  {groupDraft && <GroupDraftItem groupDraft={groupDraft} groupHome={groupHome} />
  {groupsContent}
</ul>
```

That will allow you to encapsulate the logic a little better, potentially test the standalone component (if you want), and avoid creating variables that might not be needed on every render.